lib/bloom_remit_client.rb

```diff
@@ -1,29 +1,48 @@
+# frozen_string_literal: true
```

Comment:
@ramontayag this makes string literals immutable. Strings immutability will be default in ruby 3.0 
https://wyeworks.com/blog/2015/12/1/immutable-strings-in-ruby-2-dot-3